third_party/terraform/utils/compute_instance_helpers.go

```diff
@@ -30,17 +57,87 @@ func flattenAliasIpRange(ranges []*computeBeta.AliasIpRange) []map[string]interf
 	return rangesSchema
 }
 
-func flattenScheduling(scheduling *computeBeta.Scheduling) []map[string]interface{} {
-	result := make([]map[string]interface{}, 0, 1)
+func expandScheduling(v interface{}) (*computeBeta.Scheduling, error) {
+	if v == nil {
+		return &computeBeta.Scheduling{
+			AutomaticRestart: googleapi.Bool(true),
```

Comment:
Likewise here, I think it would help to have comments to know why we're setting defaults in code